package.json

```diff
@@ -7,7 +7,9 @@
   "dependencies": {
     "@oclif/config": "^1",
     "@salesforce/command": "^3.0.3",
-    "@salesforce/core": "^2.16.3",
+    "@salesforce/core": "^2.17.0",
```

Comment:
Pulled in 2.17.0 - changes needed to isInternalUrl, etc.
1.4.0 of kit was bumped "automagically"